source/ShipInfoDisplay.cpp

```diff
@@ -169,9 +169,10 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const Depreciation &dep
 		attributeValues.push_back(Format::Number(attributes.Get("hull")));
 	}
 	attributesHeight += 20;
-	double emptyMass = ship.Mass();
+	double emptyMass = attributes.Get("mass");
```

Comment:
```suggestion
	double emptyMass = attributes.Mass();
```
Sorry, my mistake in my previous comment, Mass is a special case and stored directly, not in the map (similar to Cost()).